### PR TITLE
Data dictionary: Add column with field description

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Comment by davisagli** [comment]
What's the context for this? My first reaction is that it might make more sense to always output both the help text and description in separate columns (it's easy enough to delete a column after the data dictionary is generated if it's not desired) rather than controlling it using a task option.

File: cumulusci/cumulusci.yml [hunk 1]
```diff
@@ -178,6 +178,10 @@ tasks:
         description: Lists open pull requests in project Github repository
         class_path: cumulusci.tasks.github.PullRequests
         group: GitHub
+    github_protect_branch:
+        description: Protects master branch in project Github repository
+        class_path: cumulusci.tasks.github.protect_branch.ProtectBranch
+        group: GitHub
```
Comment (davisagli) [inline cumulusci/cumulusci.yml:184]: @jofsky accidental commit?

File: cumulusci/tasks/datadictionary.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 from distutils.version import LooseVersion
 
-
+from cumulusci.core.utils import process_bool_arg
```
Comment (davisagli) [inline cumulusci/tasks/datadictionary.py:7]: Please remove this import which you're no longer using.

And please make sure your editor runs flake8 on save, which will warn you about things like this.

File: cumulusci/tasks/tests/test_datadictionary.py [hunk 3]
```diff
@@ -628,6 +638,17 @@ def test_init_options__defaults(self):
...
+    def test_init_options__defaults(self):
+        project_config = create_project_config()
+        project_config.project__name = "Project"
+
+        task = create_task(
+            GenerateDataDictionary, {"release_prefix": "rel/"}, project_config
+        )
+
+        assert task.options["object_path"] == "Project sObject Data Dictionary.csv"
+        assert task.options["field_path"] == "Project Field Data Dictionary.csv"
```
Comment (davisagli) [inline cumulusci/tasks/tests/test_datadictionary.py:650]: You added another copy of test_init_options__defaults, which is simply replacing the prior definition. This is another example of a mistake that flake8 would tell you about.

File: cumulusci/cumulusci.yml (hunk 1 above)
Comment (jofsky) [reply cumulusci/cumulusci.yml:184]: indeed good catch.

**Pull request merged**